src/os/bluestore/BlueStore.cc

```diff
@@ -142,6 +142,18 @@ const string PREFIX_ZONED_CL_INFO = "G";  // (per-zone cleaner metadata)
 
 const string BLUESTORE_GLOBAL_STATFS_KEY = "bluestore_statfs";
 
+// Label offsets where they might be replicated. It is possible on previous versions where these offsets
+// were already used so labels won't exist there.
+const vector<uint64_t> bdev_label_offsets = {0, 1073741824, 107374182400, 1099511627776};
+
+// write a label in the first block.  always use this size.  note that
+// bluefs makes a matching assumption about the location of its
+// superblock (always the second block of the device).
+#define BDEV_LABEL_BLOCK_SIZE  4096
+
+// reserve: label (4k) + bluefs super (4k), which means we start at 8k.
+#define SUPER_RESERVED  8192
```

Comment:
The change DB_SUPER_RESERVED -> SUPER_RESERVED is very good.
It has nothing to do with DB.